Package.swift

```diff
@@ -170,6 +171,8 @@ let package = Package(
         .copy("Frontend/UserContent/UserScripts/WalletEthereumProvider.js"),
         .copy("Frontend/UserContent/UserScripts/YoutubeAdblock.js"),
         .copy("Frontend/UserContent/UserScripts/DeAMP.js"),
+        .copy("Frontend/UserContent/UserScripts/Sandboxed/ResourceDownloader.js"),
+        .copy("Frontend/UserContent/UserScripts/Sandboxed/WindowRenderHelper.js"),
```

Comment:
I didn't even realize these scripts weren't packed, can we just move them out to the root `UserScripts` folder or audit all the scripts in that folder and make sure we put the sandboxed ones here? Just quickly running through I see the `DeAMP.js` is also sandboxed but not in this folder. 

Given the rule for sandboxing a script is code based at `WKUserScript` creation and not folder based (for non-packed scripts) we could just leave them all in a flat folder for now until we move